### ISSUE TITLE
Salvage and expedition console on ruins.

### ISSUE THREAD

**Issue by Pik1ta** [space-wizards/space-station-14#33100]
Salvage and expedition console dosent work on ruins, like syndicate dropship. When trying to take a salvage disk, or grid, nothing happen.

Maybe the problem is in the components?

**Comment by Pik1ta** [comment]
I don't think it should work like that, considering that now Salvagers They don't have their own shuttle, but capturing grids and repairing them looks pretty attractive.

**Comment by deltanedas** [comment]
any similar consoles wont work because ruins arent station members

**Comment by Pik1ta** [comment]
> any similar consoles wont work because ruins arent station members

Is there any way to change this or specify an exception?

**Comment by Pik1ta** [comment]
> any similar consoles wont work because ruins arent station members

Is there any way to change this or specify an exception?

**Comment by deltanedas** [comment]
change how the
and for the love of god if you do this distinguish between a stations physical grids and grids that belong to it so events just cant happen on cargo shuttle / ruins etc, but they still keep the benefits of belonging to the station

**Comment by Pik1ta** [comment]
Soo.. its working so strange.  Maybe it's worth trying to give some ruins Shuttle component?
